sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/KubernetesClusterTests.java

```diff
@@ -29,4 +31,10 @@ public void testDeployImageFromContainerRegistryToKubernetes() throws JSchExcept
             Assertions.assertTrue(DeployImageFromContainerRegistryToKubernetes.runSample(azureResourceManager, "", ""));
         }
     }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
```

Comment:
why do we have both test and sample?

Reply:
The test is for validating that the sample code works.